src/components/alert/alert.scss

```diff
@@ -14,17 +14,25 @@
 
   .alert {
     position: absolute;
-    top: 200px;
     left: 50%;
     transform: translate(-50%, -50%);
-    max-width: 424px;
     border-radius: 8px;
     background: $color-neutral-light-snow;
     box-shadow: $shadow-box;
     overflow: hidden;
     position: relative;
   }
 
+  .alert-standard {
+    top: 200px;
+    max-width: 424px;
+  }
+
+  .alert-wide {
```

Comment:
You need align changes like this with the 'interaction', because the impact are big. Please contact your PD.